spec/core_functions/meta/accepts_content.hrx

```diff
@@ -0,0 +1,77 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#2807
+
+<===>
+================================================================================
+<===> accepts/direct-child/input.scss
+@use "sass:meta";
+
+@mixin foo() {
```

Comment:
[Use single-letter names for irrelevant placeholders](https://github.com/sass/sass-spec/blob/main/STYLE_GUIDE.md#do-use-single-letter-names-for-irrelevant-placeholders)